core/templates/dev/head/collection_editor/settings_tab/CollectionDetailsEditorDirective.js

```diff
@@ -43,13 +43,20 @@ oppia.directive('collectionDetailsEditor', [function() {
             };
           }
         );
+        $scope.LANGUAGE_LIST_FOR_SELECT = function() {
```

Comment:
perhaps this should be `$scope.getLanguageListForSelect()`? `LANGUAGE_LIST_FOR_SELECT` sounds like a constant and not a function.